DiscordChatExporter.Gui/ViewModels/RootViewModel.cs

```diff
@@ -236,6 +237,25 @@ public async void ExportChannels()
             // Notify of overall completion
             if (successfulExportCount > 0)
                 Notifications.Enqueue($"Successfully exported {successfulExportCount} channel(s)");
+
+            IPartitioner CreatePartitioner()
+            {
+                var partitionFormat = dialog.SelectedPartitionFormat;
+                var partitionLimit = dialog.PartitionLimit;
+                if (partitionLimit == null)
+                {
+                    return new NullPartitioner();
+                }
+
+                if (partitionFormat == PartitionFormat.MessageCount)
```

Comment:
Maybe `PartitionFormat` should implement an enum class pattern and have a `CreatePartitioner` method to avoid this if-else chain. However, if we don't plan on adding any more partition formats, this seems like extra work for nothing.